src/styles/_catalog.scss

```
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

```suggestion
    transition-duration: 300ms;
    transition-property: transform, color;
```
Завжди потрібно вказувати властивості, до яких буде застосовуватись transition